Update README.md (#1)
landonreed/landonreed: README.md

I'm a software developer in 🍑 Atlanta, GA (USA).

### 😄 About me 
I work on open source software for public transit agencies. These are some of the projects:

> - this
> - that
> - the other

Author: ```suggestion
- that
- this
- another
```

I'm a software developer in 🍑 Atlanta, GA (USA).

### 😄 About me 

> I work on open source software for public transit agencies, e.g.:

Author: ```suggestion
I work on open source software for public transit agencies, e.g.:
- Ragtag - volunteer work!
```

> I work on open source software for public transit agencies, e.g.:

Author: ```suggestion
I work on open source software for public transit agencies. These are some of the projects:
```

> I work on open source software for public transit agencies, e.g.,:

Author: ```suggestion
I work on open source software for public transit agencies, e.g.:
```

> I work on open source software for public transit agencies. These are some of the projects:

Author: ```suggestion
I work on open source software for public transit agencies. These are some of the projects:
- this
- that
- the other
```
